server/planning/planning/planning.py

```diff
@@ -63,6 +64,19 @@
 class PlanningService(superdesk.Service):
     """Service class for the planning model."""
 
+    def post_in_mongo(self, docs, **kwargs):
```

Comment:
Why are changes in `post_in_mongo` and `patch_in_mongo` needed?

Reply:
Because we needed it [here](https://github.com/superdesk/superdesk-core/blob/develop/superdesk/io/commands/update_ingest.py#L674).